### ISSUE TITLE
Issue rendering tables with certain markup

### ISSUE THREAD

**Issue by tobiasbunyan** [lunet-io/markdig#87]
Having problems with tables when adding a table referencing source control paths.  For example, this renders ok
```
Status|Path
------|----
Active|$/val
Active|$val
```
but add in / to the second path value
```
Status|Path
------|----
Active|$/val
Active|$/val
```
and the output goes wonky

Escaping the second $ (i.e. \\$) fixes the problem.  I'm not aware of $ being a special char in markdown?

This is experienced in the preview window of the [Markdown Editor](https://marketplace.visualstudio.com/items?itemName=MadsKristensen.MarkdownEditor) visual studio extension version 1.10.186 on Visual Studio Enterprise 2015 14.0.25431.01 Update 3.

madskristensen/MarkdownEditor#90

**Comment by Kryptos-FR** [comment]
Confirmed by https://babelmark.github.io/?text=Status%7CPath%0A------%7C----%0AActive%7C%24%2Fval%0AActive%7C%24%2Fval and seems to be markdig-only issue.

**Comment by xoofx** [comment]
Yes, the `$` is used by the extension mathematics and it look like it has a higher precedence than table pipes `|`...  when markdig is configured with advanced extension, the math extension is enabled (and Markdown Editor is using this).

I'm quite busy this week... so won't be able to look at this before next week

**Comment by xoofx** [comment]
Fixed by commit 26a565fa453be4f6c0b09b840702cb54821f6167
